etc/authordb.yaml

```
@@ -3818,6 +3854,16 @@ authors:
     family_name: Calabrese
     given_name: Daniel
     orcid: null
+  calderonj:
+    affil:
+    - TorVergata
+    - UCA 
+    - UdeA 
+    altaffil: []
+    email: jeronimo.calderon-gomez@etu.univ-cotedazur.fr
```

```suggestion
    email: jeronimo.calderon-gomez@UCA
```
We prefer not to have full email addresses here so you can say `@UCA` and it looks up the email domain from affiliation UCA.